### PR TITLE
MSP-3642: add some configmap values

### PR DESCRIPTION
**Pull request by Uburro**

### CODE REVIEW

File: helm/soperator-crds/templates/slurmcluster-crd.yaml [hunk 1]
```diff
@@ -1468,6 +1468,76 @@ spec:
                       with ssh keys for SSHD server
                     type: string
                 type: object
+              slurmConfig:
+                description: SlurmConfig represents the Slurm configuration in slurm.conf.
+                  Not all options are supported.
+                properties:
+                  completeWait:
+                    default: 0
```
Comment (rdjjke) [inline helm/soperator-crds/templates/slurmcluster-crd.yaml]: Let's set it to 5 by default

File: helm/soperator-crds/templates/slurmcluster-crd.yaml [hunk 2]
```diff
@@ -1468,6 +1468,76 @@ spec:
...
+                    description: The time to wait, in seconds, when any job is in
+                      the COMPLETING state before any additional jobs are scheduled.
+                    format: int32
+                    type: integer
+                  debugFlags:
+                    default: ""
```
Comment (rdjjke) [inline helm/soperator-crds/templates/slurmcluster-crd.yaml]: Let's set it to `Cgroup,CPU_Bind,Gres,JobComp,Priority,Script,SelectType,Steps,TraceJobs` by default

File: api/v1/slurmcluster_types.go [hunk 3]
```diff
@@ -73,6 +73,36 @@ type SlurmClusterSpec struct {
...
 	PartitionConfiguration PartitionConfiguration `json:"partitionConfiguration,omitempty"`
+
+	// SlurmConfig represents the Slurm configuration in slurm.conf. Not all options are supported.
+	//
+	// +kubebuilder:validation:Optional
+	SlurmConfig SlurmConfig `json:"slurmConfig,omitempty"`
+}
+
+// SlurmConfig represents the Slurm configuration in slurm.conf
+type SlurmConfig struct {
```
Comment (rdjjke) [inline api/v1/slurmcluster_types.go:84]: Can we add the `TaskPluginParam` with the default value `OOMKillStep,Verbose`?

Comment (Uburro) [reply api/v1/slurmcluster_types.go:84]: If we want to kill by OOM an entire step set memory.oom.group = 1 to step for OOMKillStep

**Review by rdjjke — APPROVED** [review]

**Pull request merged**